### PR TITLE
Development sample application

### PR DESCRIPTION
**Pull request by looser990**

### CODE REVIEW

File: ap_ruby_sdk/lib/ap_ruby_sdk.rb [hunk 1]
```diff
@@ -56,10 +56,10 @@ def self.request(method, url, api_key, params={}, headers={})
 
     request_opts = {
         :headers => request_headers(api_key).update(headers),
-        :method => method, :open_timeout => 30,
-        :payload => payload, :url => url, :timeout => 80
+        :method => method,
+        :payload => payload,
+        :url => url
```
Comment (nbozic) [inline ap_ruby_sdk/lib/ap_ruby_sdk.rb]: Think it is useful to leave timeouts, not to get stuck and bombarded with some huge response (just for the case). What are defaults in open_timeouts and timeout?

File: sample-application/app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
```
Comment (nbozic) [inline sample-application/app/assets/stylesheets/application.css]: Do we need all these generated placeholder files?

**Comment by nbozic** [comment]
As a general remark, what is absolute minimum number of files here. It looks like lot of things are generated and maybe we do not need all of them, can we do some cleanup not to bring noise?

**Comment by looser990** [comment]
Done clean-up and added timeouts. Also updated .gitignore.

**Pull request merged**